.github/workflows/build.yml

```diff
@@ -22,3 +22,6 @@ jobs:
       - name: Install plugin
```

Comment:
You are installing our whole repository with custom plugins, not just one unnamed "plugin".
*Install plugins* will be OK.